docs/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "attractions-docs",
+  "description": "The documentation for the Attractions UI kit",
+  "license": "MIT",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "sapper dev",
+    "build": "sapper build --legacy",
+    "export": "sapper export --legacy",
+    "start": "node __sapper__/build"
+  },
+  "dependencies": {
+    "compression": "^1.7.1",
+    "polka": "next",
+    "sirv": "^0.4.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/runtime": "^7.0.0",
+    "@rollup/plugin-alias": "^3.1.1",
+    "@rollup/plugin-babel": "^5.0.0",
+    "@rollup/plugin-commonjs": "^12.0.0",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "@rollup/plugin-replace": "^2.2.0",
+    "attractions": "^2.0.1-alpha",
```

Comment:
Will it not be linked directly to it through `../attractions`? It's not like `docs` is a package that will get released to npm, this package.json is there just for the dependencies, right?

Reply:
I thought so as well, but upon importing from `../../attractions`, Sapper complains that the imported things are not SSR components, so I decided to install it as a dependency.

As I was writing this, I realized that we don't really need SSR, since we will be exporting the website statically anyway. In that case we should try to use the existing version (we can even alias `attractions` to `~/attractions` for convenience). If the exporting works, we can simply turn off SSR

Reply:
What kind of complaints, and why are they not SSR components?

Reply:
It's just a thing in Sapper, if you want to use external components with SSR, you must install libraries to dev dependencies

Reply:
I'll let you deal with that, but I believe we need to link directly instead of use it as a dependency because when editing a component and its docs together, the docs will not see the update in the component